### PR TITLE
56 range allow floats

### PR DESCRIPTION
**Pull request by fxnitsch**

### CODE REVIEW

File: scengen/generator.py [hunk 1]
```diff
@@ -77,6 +84,22 @@ def generate_scenario(options: dict) -> None:
     write_yaml(scenario, options["scenario_path"])
 
 
+def _get_number_of_agents_to_create(agent_count: Union[List[Any], Any], options: dict) -> int:
+    """
+    Returns an integer number from field `agent_count`
+    Accepts float values by rounding to integer, but raises warning to notify user about wrong type, in all other cases an Error is raised
```
Comment (dlr-cjs) [inline scengen/generator.py:90]: Change last port of doctring to
"...for any other datatype an Error is raised"

File: scengen/generator.py [hunk 2]
```diff
@@ -97,17 +120,23 @@ def _get_random_seed(defaults: dict) -> int:
...
-    In option `RANGE_IDENTIFIER`, `allow_negative` is True as default but can limit allowed range to values >=0.
+    Returns value stored in `input_value` based on the user specification 'RANGE_INT_IDENTIFIER',
+    'RANGE_INT_IDENTIFIER', 'CHOOSE_IDENTIFIER', 'PICKFILE_IDENTIFIER' or else just `input_value`.
+    In range options, `allow_negative` is True as default but can limit allowed range to values >=0.
     In option `PICKFILE_IDENTIFIER`, `options[CreateOptions.DIRECTORY]` is used to get files: paths relative to scenario
     """
     if isinstance(input_value, str):
-        if RANGE_IDENTIFIER in input_value.lower():
-            input_range = _digest_range(input_value)
+        if RANGE_IDENTIFIER_DEPRECATED in input_value.lower():
```
Comment (dlr-cjs) [inline scengen/generator.py:129]: nice!

**Review by dlr-cjs — APPROVED** [review]
Good changes!

File: scengen/generator.py (hunk 1 above)
Comment (fxnitsch) [reply scengen/generator.py]: Done

**Pull request merged**